src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -221,11 +221,13 @@ private void checkAnnotationIndentation(DetailAST atNode,
         while (firstNodesOnLines.size() > 1) {
             final DetailAST node = itr.next();
 
-            if (node.getLineNo() < lastAnnotationLine
-                    || node.getLineNo() == lastAnnotationLine) {
+            if (node.getLineNo() <= lastAnnotationLine) {
                 final DetailAST parentNode = node.getParent();
+                final boolean isCurrentNodeCloseAnnotationAloneInLine =
+                        node.getLineNo() == lastAnnotationLine && node == lastAnnotationNode;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use equals() to compare object references.](https://www.codacy.com/app/checkstyle/checkstyle/file/1008404572/issues/source?bid=2193633&fileBranchId=2739117#l227)